src/lib.rs

```diff
@@ -44,6 +44,7 @@ pub fn run(args: Args) -> Result<()> {
         }
     }
     config.update_from_args(&args);
+    config.parse_token_files();
```

Comment:
@tessus 

> I just want to make sure that even if the file is set in the config file, the token can be overridden via the command line argument -a.

I think changing the order of these calls will do it:

```suggestion
    config.parse_token_files();
    config.update_from_args(&args);
```